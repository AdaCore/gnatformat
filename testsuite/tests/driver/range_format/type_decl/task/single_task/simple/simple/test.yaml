description: SingleTaskDecl formatting for line selection.
driver: gnatformat
args:
  [
    "test.ads",
    "--range-format",
    "-SL 138",
    "-EL 139",
    "--pipe",
  ]
