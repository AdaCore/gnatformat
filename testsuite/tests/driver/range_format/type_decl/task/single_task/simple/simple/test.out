test.ads(<SingleTaskDecl ["User"] test.ads:138:4-138:14>) - 138:1-138:14
^
   task User;$

